src/core/util/sys_vars.h

```diff
@@ -793,10 +793,10 @@ extern mce_sys_var &safe_mce_sys();
 #endif
 #define MCE_DEFAULT_CQ_MODERATION_COUNT            (48)
 #define MCE_DEFAULT_CQ_MODERATION_PERIOD_USEC      (50)
-#define MCE_DEFAULT_CQ_AIM_MAX_COUNT               (560)
-#define MCE_DEFAULT_CQ_AIM_MAX_PERIOD_USEC         (250)
-#define MCE_DEFAULT_CQ_AIM_INTERVAL_MSEC           (250)
-#define MCE_DEFAULT_CQ_AIM_INTERRUPTS_RATE_PER_SEC (5000)
+#define MCE_DEFAULT_CQ_AIM_MAX_COUNT               (500)
+#define MCE_DEFAULT_CQ_AIM_MAX_PERIOD_USEC         (1000)
+#define MCE_DEFAULT_CQ_AIM_INTERVAL_MSEC           (1000)
+#define MCE_DEFAULT_CQ_AIM_INTERRUPTS_RATE_PER_SEC (1000)
```

Comment:
what is the motivation for those values?
i try to understand from the code but I don't